build/repo.targets

```diff
@@ -13,5 +13,8 @@
     </ItemGroup>
     <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="TestProjects" Properties="FakeProperty=%(Iteration.Identity)" />
   </Target>
-</Project>
 
+  <Target Name="EnableTestLogging" BeforeTargets="_ExecuteTestAssemblies" Condition="'$(ASPNETCORE_TEST_LOG_DIR)' == '' AND '$(DisableFTestLogging)' != 'true'">
```

Comment:
Looks like test invocation works differently for ./build.ps1 and dotnet test. The workaround is to add the custom target that executes before _ExecuteTestAssemblies and VSTest.

Reply:
😬 this is like dependency on pubternal API. The underscore convention in MSBuild is meant to indicate we make no promises about removing or changing this target in the future.

Reply:
Right but I couldn't find a better target.